source/CardGames.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 cardImages = []
-values = [11,2,3,4,5,6,7,8,9,10,10,10]
+values = [11,2,3,4,5,6,7,8,9,10,10,10,10]
 suits = ["Spades", "Clubs", "Hearts", "Diamonds"]
```

Comment:
Good I fixed this as well on my pull request.

Reply:
awesome! glad this is resolved.